charts/simple-app/values.yaml

```diff
@@ -228,10 +260,10 @@ affinity: {}
 tests:
   connection:
     # -- The command used to trigger the test.
-    command: ['curl']
+    command: ['curl', '--retry-connrefused', '--retry', '5']
```

Comment:
do you want to add a retry delay?

Reply:
There is an automatic doubling backoff delay - so it goes 1s, 2s, 4s, 8s, 16s...